index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  </head>
+
+  <body>
+    
+    <div id="header">
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="">      
+     <div id="hehe"> 
+      <nav id="nav-bar">
+      <ul>
+        <li>
+          <a href="#features" class="nav-link"> Features</a>
+        </li>
+        <li>
+          <a href="#how-it-works" class="nav-link"> How It Works</a>
+        </li>
+        <li>
+          <a href="#pricing" class="nav-link"> Pricing</a>
+        </li>
+      </ul>
+      </nav>
+      </div>
+    </div>
+    <div id="above">
+      <h2>Handcrafted, home-made masterpieces</h2>
+      <form id="form" action="https://www.freecodecamp.com/email-submit">
+        <input type="email" name="email"  id="email" placeholder="Enter your email address" required>
+        <br>
+        <input type="submit" id="submit" value="Get Started">
+
```

Comment:
put these forms in a div

Reply:
Good job! but It's highly recommended to be more polite. In here you can do suggestion instead of orders. And kindly make sure to explain the **Why** behind your suggestion.

Reply:
### Please,  put the form in a div. 
**Its symbolic purpose is to divide a section/portion in the page so that a separate style can be added or applied to it.**